make/cluster.sh

```diff
@@ -179,6 +179,13 @@ setup_kind() {
       --name "$kind_cluster_name"
   fi
 
+  # Sleep to wait for cluster-info to be up to date
+  # TODO (irbekrm): loop and repeatedly check for the info to become available instead of sleeping
```

Comment:
This might have been cause of some of the flakes we were also swallowing the error if the ip ranges were empty https://github.com/cert-manager/cert-manager/blob/master/make/cluster.sh#L211